Cargo.toml

```diff
@@ -34,10 +34,10 @@ heck = "0.4.0"
 
 [dependencies]
 edgeql-parser = {git = "https://github.com/edgedb/edgedb"}
-edgedb-protocol = {git = "https://github.com/edgedb/edgedb-rust/", features=["all-types"]}
-edgedb-derive = {git = "https://github.com/edgedb/edgedb-rust/"}
-edgedb-errors = {git = "https://github.com/edgedb/edgedb-rust/"}
-edgedb-tokio = {git = "https://github.com/edgedb/edgedb-rust/", features=["admin_socket", "unstable"]}
+edgedb-protocol = {git = "https://github.com/edgedb/edgedb-rust/", features=["all-types"], rev = "cf5043a"}
+edgedb-derive = {git = "https://github.com/edgedb/edgedb-rust/", rev = "cf5043a"}
+edgedb-errors = {git = "https://github.com/edgedb/edgedb-rust/", rev = "cf5043a"}
+edgedb-tokio = {git = "https://github.com/edgedb/edgedb-rust/", features=["admin_socket", "unstable"], rev = "cf5043a"}
```

Comment:
we should update `edgedb-rust`

Reply:
This dependency here uses the last commit from the repo or whatever ref is saved in `Cargo.lock`.

So I think you wanted to do `cargo update edgedb-tokio` here, and not pin these deps on one specific commit.